contrib/hashicorp/vault/vault.go

```diff
@@ -0,0 +1,71 @@
+// Package vault contains functions to construct an *http.Client that will
```

Comment:
```suggestion
// Package vault contains functions to construct or augment an http.Client that will
```
It feels a bit weird to to me to use a pointer in documentation. I've not seen that before.

Reply:
They're gone.